tests/package/test_install_uninstall.py

```diff
@@ -5,23 +5,60 @@
 :caseautomation: Automated
 """
 import pytest
-from virtwho import logger
+from virtwho import VIRTWHO_PKG
+from virtwho.base import package_check, package_install, package_uninstall
+from virtwho.base import wget_download, random_string
+from virtwho.testing import virtwho_pacakge_url
 
 
 class TestInstallUninstall:
     @pytest.mark.tier1
-    def test_install_uninstall_by_yum(self):
-        """Just a demo
+    def test_install_uninstall_by_yum(self, ssh_host):
+        """Test virt-who install/uninstall by yum
 
-        :title: VIRT-WHO: test install uninstall by yum
-        :id: d2b53819-e876-4595-8407-549eb8ef1bdb
+        :title: virt-who: package: install/uninstall by yum
+        :id: 4f784cd4-ad46-4457-a20f-7cf07ba44d9e
         :caseimportance: High
         :tags: tier1
         :customerscenario: false
         :upstream: no
         :steps:
-            1.
+            1. uninstall virt-who by #yum remove virt-who
+            2. install virt-who by #yum install virt-who
         :expectedresults:
-            1.
+            1. virt-who can be remove and reinstall successfully.
         """
-        logger.info("Succeeded to run the 'test_install_uninstall'")
+        package_uninstall(ssh_host, 'virt-who')
+        assert package_check(ssh_host, 'virt-who') is False
+        package_install(ssh_host, 'virt-who')
+        assert package_check(ssh_host, 'virt-who') == VIRTWHO_PKG
+
+    @pytest.mark.tier1
+    def test_install_uninstall_by_rpm(self, ssh_host):
+        """Test virt-who install/uninstall by rpm
+
+        :title: virt-who: package: install/uninstall by rpm
+        :id: 4f784cd4-ad46-4457-a20f-7cf07ba44d9e
```

Comment:
Same uuid with test_install_uninstall_by_yum

Reply:
Changed it.